ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -197,3 +197,12 @@ def logistic(loc=0.0, scale=1.0, size=None):
     u = ivy.random_uniform(low=0.0, high=0.0, shape=size, dtype="float64")
     x = loc + scale * ivy.log(u / (1 - u))
     return x
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def zipf(a, size=None):
+    if a <= 1:
+        return 0
+    x = ivy.np.random.zipf(a, size=size)
```

Comment:
You cannot use `ivy.np` in the frontends since `ivy.np` points to the `numpy` package and so any native frameworks cannot be imported/used in the frontends ever. We're just calling the `np.random.zipf` function directly here whereas the point of this frontend function is to implement it using `ivy` functions